assets/js/googlesitekit/api/index.js

```diff
@@ -345,6 +345,8 @@ const API = {
 	set,
 	setUsingCache,
 	usingCache,
+	getItem,
+	setItem,
```

Comment:
This isn't strictly necessary but could be good to include here. Other uses import the functions from `api/cache`. These functions are used for other values than API cache specifically, so it could be we want to expose them differently (if at all). Thoughts @tofumatt ?